jwst/pixel_replace/pixel_replace.py

```diff
@@ -326,6 +333,104 @@ def fit_profile(self, model):
 
         return model_replaced
 
+    def mingrad(self, model):
+        """
+        Minimum gradient replacement method; test the gradient along the spatial
+        and spectral axes using immediately adjacent pixels.  Pick whichever dimension
+        has the minimum absolute gradient and replace the missing pixel with the average
+        of the two adjacent pixels along that dimension.
+
+        This aims to make the process extremely local; near point sources it should do
+        the replacement along the spectral axis avoiding sampling issues, while near bright
+        extended emission line the replacement should be along the spatial axis.  May still
+        be suboptimal near bright emission lines from unresolved point sources.
+
+        Does not attempt any replacement if a NaN value is bordered by another NaN value
+        along a given axis.
+
+        Parameters
+        ----------
+        model : DataModel
+            Either the input to the pixel_replace step in the
+            case of DataModels containing only one 2D spectrum,
+            or a single 2D spectrum from the input DataModel
+            containting multiple spectra (i.e. MultiSlitModel).
+            Requires data and dq attributes.
+
+        Returns
+        -------
+        model_replaced : DataModel
+            DataModel with flagged bad pixels now flagged with
+            TO-BE-DETERMINED and holding a flux value estimated
```

Comment:
Don't we now have a DQ name that's been appropriated for this? If so, the comment could be updated.

Reply:
Yes, FLUX_ESTIMATED.  I'll make this change here and in the original code for the default method.